Makefile

```diff
@@ -13,6 +13,8 @@ ETCDIR ?= ${DESTDIR}/etc
 ETCDIR_OCID ?= ${ETCDIR}/ocid
 BUILDTAGS := selinux seccomp $(shell hack/btrfs_tag.sh) $(shell hack/libdm_tag.sh)
 BASHINSTALLDIR=${PREFIX}/share/bash-completion/completions
+GOBINDIR := $(word 1,$(subst :, ,$(GOPATH)))
+PATH := $(GOBINDIR)/bin:$(PATH)
```

Comment:
It's just a personal preference, I guess, but I don't like modifying `PATH` and would rather be explicit and use `$(GOBINDIR)/ocid` instead (since that's what we're installing in `make install`)

Thanks for fixing this :+1: I evidently don't use a multi-part GOPATH so didn't have this problem